.github/workflows/cd-android.yaml

```
@@ -0,0 +1,51 @@
+name: CD Package Android
+
+permissions:
+  contents: write # to update the release notes
+  pull-requests: write # to post apk link comment
+
+on:
+  pull_request:
```

Any chance we’d ever want to manually retrigger a build? 🤔 

```suggestion
on:
  pull_request:
  workflow_dispatch:
```